saleor/account/migrations/0057_clear_user_addresses.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.12 on 2022-02-15 14:18
+
+from django.db import migrations
+from django.db.models import Count
+
+
+def clear_addresses(apps, schema_editor):
+    User = apps.get_model("account", "User")
+    Address = apps.get_model("account", "Address")
+
+    users = User.objects.annotate(address_count=Count("addresses")).filter(
+        address_count__gt=100
```

Comment:
we use 100 here and below, maybe worth extracting it to meaningfull const like `USER_ADDRESS_LIMIT`